### ISSUE TITLE
Add aria-label to section elements inside of TableToolbar carbon react component

### ISSUE THREAD

**Issue by siddarthpatel** [IBM/carbon-components-react#1274]
Trying to add an aria-label to section elements inside of TableToolbar carbon react component, for improving the accessibility features. There are no aria-labels to section elements currently.

The issue related to the TableToolbar component inside of DataTable carbon react component. Here's a link to the issue that includes a brief description and screenshots.
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/13693

**Comment by joshblack** [comment]
Hi there @siddarthpatel! 👋 Thanks so much for taking the time to make this issue 😄 

For `TableToolbar`, you can actually pass in any props you want and the component should accept them. The following should be valid:

```jsx
<TableToolbar aria-label="My aria label">
  {/* ... */}
</TableToolbar>
```

Is the suggestion here that we should supply a default aria-label for the `TableToolbar`? Or that one can't add in the `aria-label`, if needed?

**Comment by siddarthpatel** [comment]
@joshblack Yes that works perfectly. Thank you 👍

**Comment by joshblack** [comment]
Awesome! Glad to hear 😄 Going to close this for now, feel free to make another issue if this ends up not working 👍
